src/decorations.ts

```diff
@@ -16,6 +16,7 @@ export function failingItName() {
         contentText: '●',
       },
     },
+    rangeBehavior: DecorationRangeBehavior.ClosedClosed,
```

Comment:
so closed they wrote it twice

Reply:
It's a terrible name, and unfortunately they haven't added the legible [alias for it](https://github.com/Microsoft/vscode/blob/114adf703be6203fba2da696aeac439449241ac9/src/vs/editor/common/editorCommon.ts#L899) to the enum:
```
export enum TrackedRangeStickiness {
	AlwaysGrowsWhenTypingAtEdges = 0,
	NeverGrowsWhenTypingAtEdges = 1,
	GrowsOnlyWhenTypingBefore = 2,
	GrowsOnlyWhenTypingAfter = 3,
}
```

The default is `OpenOpen` or `AlwaysGrowsWhenTypingAtEdges`.